osi_sensorviewconfiguration.proto

```diff
@@ -412,40 +384,23 @@ message RadarSensorViewConfiguration
     // is_greater_than_or_equal_to: 0
     // \endrules
     //
-    optional double emitter_frequency = 9;
+    optional double emitter_frequency = 6;
 
     // This represents the TX antenna diagram
     //
-    repeated AntennaDiagramEntry tx_antenna_diagram = 10;
+    repeated SpatialSignalGain tx_antenna_diagram = 7;
 
     // This represents the RX antenna diagram
     //
-    repeated AntennaDiagramEntry rx_antenna_diagram = 11;
-
+    repeated SpatialSignalGain rx_antenna_diagram = 8;
+    
+    // Emitter power.
     //
-    // \brief The radar antenna diagram.
+    // emitter_power must be considered together with tx_antenna_diagram for absolute power calculation. 
     //
-    // \note Rotation is defined analog Spherical3d
```

Comment:
Then we should add this note it in the osi_common, where the message is defined. What is not completly clear is the note itself, because we dont have the type Spherical3d here and one dimension less. Therefore the rotation can't be analog to Spherical3d. What do you think, @thomassedlmayer ?

Reply:
Yes, it should be added in osi_common. I just added the note to the removed line.
You're right, we have one dimension less. But we should still determine the direction of the angles, right? E.g. right hand rule